src/dynamic_boundary_conditions/hirds_rainfall_data_to_db.py

```diff
@@ -116,9 +116,9 @@ def add_rainfall_data_to_db(engine, site_id: str, idf: bool):
     site_data = rainfall_data_from_hirds.get_data_from_hirds(site_id, idf)
     layout_structure = rainfall_data_from_hirds.get_layout_structure_of_data(site_data)
 
-    for (skip_rows, rcp, time_period) in layout_structure:
+    for (skip_rows, rcp, time_period, category) in layout_structure:
         rain_data = rainfall_data_from_hirds.convert_to_tabular_data(
-            site_data, site_id, skip_rows=skip_rows, rcp=rcp, time_period=time_period)
+            site_data, site_id, skip_rows=skip_rows, rcp=rcp, time_period=time_period, category=category)
```

Comment:
you could make all of these normal parameters ie `skip_rows, rcp` instead of `skip_rows=skip_rows, rcp=rcp` since the names are the same so it's not adding much and there is no default parameters here. Or maybe we could make a layout structure `dataclass` or `namedtuple` since these 4 always appear together. @rosepearson what are your thoughts on this?

Reply:
changed to normal parameters for now.

Reply:
I thought about it a bit, and `namedtuple` will fit in better than `dataclass` since you are already using these 4 parameters as a tuple in other places.

You create a named tuple class with type hints and then import it where needed.
```python
class LayoutStructure(NamedTuple):
    skip_rows: int
    rcp: float
    time_period: str
    catergory: str
```

and then everything can operate on `LayoutStructure`

Reply:
or whichever name you think is better

Reply:
Done.